sphinx/users/comlinetools/conversion.rst

```diff
@@ -196,3 +196,30 @@ name pattern, then the other must be included too.  The only exception is if
       bfconvert -noflat -pyramid-resolutions 4 -pyramid-scale 2 /path/to/input out.ome.tiff
 
     .. versionadded:: 6.0.0
+
+.. option:: -cache
+
+    This option will cache the initialized reader under the same directory as 
+    the input file after initialization::
+
+      bfconvert -cache /path/to/input output.ome.tiff
+
+    .. versionadded:: 6.2.0
+
+.. option:: -cache-dir DIRECTORY
+
+    This option is to be used in conjunction with :option:`-bigtiff`. When used 
```

Comment:
Should this rather be :

```suggestion
    This option is to be used in conjunction with :option:`-cache`. When used 
```
 If so, the `showinf` documentation page might need to be updated accordingly?